worlds/wargroove2/Client.py

```diff
@@ -0,0 +1,858 @@
+            if self.ctx.has_death_link:
+                death_link_send_file = os.path.join(self.ctx.game_communication_path, "deathLinkSend")
+                if os.path.exists(death_link_send_file):
+                    os.remove(death_link_send_file)
+                self.output(f"Deathlink enabled.")
+            else:
+                death_link_receive_file = os.path.join(self.ctx.game_communication_path, "deathLinkReceive")
+                if os.path.exists(death_link_receive_file):
+                    os.remove(death_link_receive_file)
+                self.output(f"Deathlink disabled.")
+
+    def _cmd_resync(self):
+        """Manually trigger a resync."""
+        self.output(f"Syncing items.")
+        self.ctx.syncing = True
+
+    def _cmd_commander(self, *commander_name: Iterable[str]):
+
+        """Set the current commander to the given commander."""
+        if commander_name:
+            self.ctx.set_commander(' '.join(commander_name[0]))
+        else:
+            if self.ctx.can_choose_commander:
+                commanders = self.ctx.get_commanders()
+                wg2_logger.info('Unlocked commanders: ' +
+                                ', '.join((commander.name for commander, unlocked in commanders if unlocked)))
+                wg2_logger.info('Locked commanders: ' +
+                                ', '.join((commander.name for commander, unlocked in commanders if not unlocked)))
+            else:
+                wg2_logger.error('Cannot set commanders in this game mode.')
+
+
+class Wargroove2Context(CommonContext):
+    command_processor = Wargroove2ClientCommandProcessor
+    game = "Wargroove 2"
+    items_handling = 0b111  # full remote
+    current_commander: CommanderData = faction_table["Starter"][0]
+    can_choose_commander: bool = False
+    commander_defense_boost_multiplier: int = 0
+    income_boost_multiplier: int = 0
+    starting_groove_multiplier: int = 0
+    has_death_link: bool = False
+    has_sacrifice_summon: bool = True
+    victory_locations: int = 1
+    objective_locations: int = 1
+    final_levels: int = 1
+    level_shuffle_seed: int = 0
+    slot_data: dict[str, Any]
+    stored_finale_key: str = ""
+    player_stored_units_key: str = ""
+    ai_stored_units_key: str = ""
+    faction_item_ids = {
+        'Starter': 0,
+        'Cherrystone': 252034,
+        'Felheim': 252035,
+        'Floran': 252036,
+        'Heavensong': 252037,
+        'Requiem': 252038,
+        'Pirate': 252039,
+        'Faahri': 252040
+    }
+    buff_item_ids = {
+        'Income Boost': 252032,
+        'Commander Defense Boost': 252033,
+        'Groove Boost': 252041,
+    }
+
+    def __init__(self, server_address, password):
+        super(Wargroove2Context, self).__init__(server_address, password)
+        self.send_index = 0
+        self.syncing = False
+        self.awaiting_bridge = False
+
+        options = get_settings()
```

Comment:
```suggestion
```
This appears to be unused. You can also get rid of the `get_settings` import since it's only used here.